cmd/newtemplates/templates.go

```diff
@@ -0,0 +1,74 @@
+// Package templates provides the templates for the k6 new command
+package templates
```

Comment:
I have no very strong opinion here, but I think I'd prefer if we can rename the folder to match the package name (i.e. just `templates`) or to a bit more concise/generic name, because I'm not sure if adding the command prefix to the folder really brings any value.

Let's see what others think - cc/ @oleiade

Reply:
I agree with you @joanlopez 👍🏻  renaming the folder to just `templates` would make sense here, considering it's the only command using templates so far 🙇🏻

Reply:
My text editor kills highlighting/IntelliSense for all files inside the folder if it is called `templates` 

I'll try to fix that! But yeah, I have renamed the folder.